pyproject.toml

```diff
@@ -26,6 +27,8 @@ dependencies = [
     'rouge_score >= 0.1.2',
     'sentence_transformers >= 2',
     'sentencepiece>=0.1.95',
+    'sudachidict_core',
+    'sudachipy',
```

Comment:
Building `sudachipy` requires a Rust compiler. Could you add that to the README or move the requirement into optional?

Reply:
Or we can create a new section like `ja`

Reply:
I put them in the [optional] dependency.